health/health.d/whoisquery.conf

```diff
@@ -8,6 +8,5 @@ component: WHOIS
     units: seconds
     every: 60s
      warn: $this < $days_until_expiration_warning*24*60*60
-     crit: $this < $days_until_expiration_critical*24*60*60
```

Comment:
This is needed. `days_until_expiration_critical` is the collector configuration option as it is now. I find it strange that it is not just a value in the alarm but a chart variable but 🤷‍♂️ Need to change the collector first.

Reply:
fair enough. Will change it.